packages/pharos/src/components/toast/pharos-toaster.ts

```diff
@@ -51,7 +53,7 @@ export class PharosToaster extends PharosElement {
   }
 
   private _getToastID(id: string | null) {
-    return id || DEFAULT_ID;
+    return id || DEFAULT_ID + uuidv4();
```

Comment:
you could add a delimiter like `_` between the default_id and uuid. to ultimately get an id `toast_123` instead of `toast123`

but a very minor suggestion. i'll leave that up to you